docs/01-overview/02-running-locally.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Running Python actors locally
```

Comment:
Yeah, I'm not a fan of this title either to be honest, but it's the most appropriate to what's on the page. I think keeping the "running locally" in the title nicely leads the users to the idea that you can run the actors also on the platform, and that running locally is not the only option.

Reply:
I think that it's actually preferable that they first try to write the actors on the platform, because then they discover all the other features that the platform offers, and only then when they get serious about it, they try to develop them locally and push them via the CLI.